layouts/partials/paige/dark.css

```diff
@@ -6,7 +6,7 @@
 /* LineLink */ .chroma .lnlinks { outline: none; text-decoration: none; color: inherit }
 /* LineTableTD */ .chroma .lntd { vertical-align: top; padding: 0; margin: 0; border: 0; }
 /* LineTable */ .chroma .lntable { border-spacing: 0; padding: 0; margin: 0; border: 0; }
-/* LineHighlight */ .chroma .hl { background-color: #ffffcc }
+/* LineHighlight */ .chroma .hl { background-color: #ffffcc1f }
```

Comment:
From what I understand, this isn't a valid hex color. Can you link to an example what color this is?

Reply:
I adjusted the slider under Firefox's developer tools.

Reply:
It's not a valid color: https://www.color-hex.com/color/ffffcc1f

Reply:
I updated the color - it has a valid hex code and is easy on the eyes